libqtile/backend/wayland/qw/view.h

```diff
@@ -65,6 +65,7 @@ struct qw_view {
     char *title;
     char *app_id;
     struct wlr_scene_tree *content_tree; // Scene tree holding the view's content
+    struct wlr_output *current_output;   // used for stacking order with views on same output
```

Comment:
hmmm wlr_surface has a list of outputs, can't we just use this? https://kennylevinsen.pages.freedesktop.org/wlroots/wlr/types/wlr_compositor.h.html#struct-wlr_surface

I don't like this bookkeeping

Reply:
No problem. I didn't realise there was a list of surfaces. That's even better.